tests/e2e/default/enketo/pregnancy.wdio-spec.js

```diff
@@ -97,12 +97,12 @@ describe('Pregnancy registration', () => {
 
   });
 
-  it('Should verify that all tasks related with the high risk pregnancy were created', async () => {
+  it.skip('Should verify that all tasks related with the high risk pregnancy were created', async () => {
     const tasksTitles = ['Health facility ANC reminder', 'Danger sign follow up', 'Pregnancy home visit'];
 
     await commonPage.goToTasks();
     const tasks = await tasksPage.getTasks();    
-    expect(tasks.length).to.equal(3);
+    expect(tasks.length).to.equal(3); //TODO: investigate why 'Pregnancy home visit' task is not being generated 
```

Comment:
This comment is not needed because the test is working fine now. Can you please remove it?